mathjax3-ts/output/chtml/TeX.ts

```diff
@@ -0,0 +1,57 @@
+/*************************************************************
+ *
+ *  Copyright (c) 2017 The MathJax Consortium
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+/**
+ * @fileoverview  The TeX font parameters as an object
+ *
+ * @author dpvc@mathjax.org (Davide Cervone)
+ */
+
+export const TexFontParams = {
+    x_height:         .442,
+    quad:             1,
+    num1:             .676508,
+    num2:             .393732,
+    num3:             .44373,
+    denom1:           .685951,
+    denom2:           .344841,
+    sup1:             .412892,
+    sup2:             .362892,
+    sup3:             .288888,
+    sub1:             .15,
+    sub2:             .247217,
+    sup_drop:         .386108,
+    sub_drop:         .05,
+    delim1:          2.39,
+    delim2:          1.0,
+    axis_height:      .25,
+    rule_thickness:   .06,
+    big_op_spacing1:  .111111,
+    big_op_spacing2:  .166666,
+    big_op_spacing3:  .2,
+    big_op_spacing4:  .45, // .6,  // better spacing for under arrows and braces
```

Comment:
Is that comment still relevant? Should this entry be parameterised then?

Reply:
The original .6 is from the TeX fonts themselves, and the comment is to mention that we are using a different value from TeX.  Since I haven't done the munderover element yet, I'm not sure if it is still needed, but it is from the v2 CHTML output, so I suspect it will be.

I'm not sure what you mean by parameterizing it, here.

Reply:
I.e., have a parameter UNDER_ARROW_SPACING that can be changed whenever necessary.
Not that it is necessary.

Reply:
It shouldn't need to change.  The comment is just indicating that it is not the same as the original value in the TeX fonts.  My feeling is that this *is* already parameterized, as `big_op_spacing4` (the TeX name for this font parameter), and this is where you would change it if you wanted a different value.  I'm not sure what you gain by having another name for it (and why you wouldn't do the same for all the others if you did).